src/components/ComputeModal.js

```diff
@@ -1051,7 +1051,8 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
             value: computeConfig.autopauseThreshold,
             hidden: !isAutopauseEnabled(computeConfig.autopauseThreshold),
             tooltip: !isAutopauseEnabled(computeConfig.autopauseThreshold) ? 'Autopause must be enabled to configure pause time.' : undefined,
-            onChange: updateComputeConfig('autopauseThreshold')
+            onChange: updateComputeConfig('autopauseThreshold'),
+            'aria-label': 'Minutes of inactivity before autopausing'
```

Comment:
This spinner had no label. Normally the text above would be linked as the label, but it's a checkbox, so a bit complicated.

Reply:
> Normally the text above would be linked as the label

Which text? Asking for my edification.

Reply:
As an example, look further down in this file for "Persistent Disk size" https://github.com/DataBiosphere/terra-ui/blob/51d9c6042d74261b3f6bfa1576ed4d1dbf020e2b/src/components/ComputeModal.js#L1649. The label there is linked to the NumberInput below it.